internal/options/kube.go

```diff
@@ -21,8 +19,13 @@ type kubeOpts struct {
 	config    *rest.Config
 }
 
+// TODO: Remove k8sControlPlaneUrl
```

Comment:
`TODO`s are something I really hate in code: we can ignore it since we're already tracking with #71 :)